packages/esm-patient-orders-app/src/order-basket/order-basket.workspace.tsx

```diff
@@ -33,6 +33,7 @@ const OrderBasket: React.FC<DefaultWorkspaceProps> = ({
     error: errorFetchingEncounterUuid,
     mutate: mutateEncounterUuid,
   } = useOrderEncounter(patientUuid);
+  const [savingOrdersLoading, setSavingOrdersLoading] = useState(false);
```

Comment:
`isSavingOrders` reads clearer to me here:

```suggestion
  const [isSavingOrders, setIsSavingOrders] = useState(false);
```